src/main/java/org/openrewrite/java/testing/jmockit/JMockitBlockRewriter.java

```diff
@@ -172,6 +178,9 @@ private void rewriteResult(J.MethodInvocation invocation, List<Expression> resul
             return;
         }
         visitor.maybeAddImport(MOCKITO_IMPORT_FQN_PREFX, "when");
+        if (this.blockType == NonStrictExpectations) {
+            visitor.maybeAddImport(MOCKITO_IMPORT_FQN_PREFX, "*");
```

Comment:
What's the thought behind this wildcard import?

Reply:
I'm hitting an issue with the framework, where when I add lenient, the when keyword static reference can't be found. I've put in a workaround and added comments now. This was just a first draft attempt.

Reply:
No problem, if you want to keep working on it we can revisit when you move the PR out of Draft.

Reply:
Yes - all ready now. I think I am pretty much done with this PR